### DISCUSSION TITLE
Fully turbulent RANS blow-up on very simple case

### DISCUSSION THREAD

**Discussion by ghost** [su2code/SU2#2177] [General]
Dear all,

I was running steady RANS simulations of the transonic flow around a very simple grid and geometry: an isolated elliptic-planform wing (half model, zero sideslip angle). Wing sections are NACA0012 airfoils with closed t.e., and the mesh is made of hexahedra.
No problems at angles of attack different from zero. Nevertheless, I was unable to obtain a solution at zero incidence.
The case at zero angle of attack runs with no difficulty in inviscid conditions (Euler simulation), in viscous and laminar flow, and even with turbulence but in free transition. But when I go for fully turbulent RANS, the simulation blows up.
For the same grid and flow conditions, no difficulty was ever encountered using other flow solvers, either structured or unstructured.
Here below I provide a link to the mesh and configuration files:

https://cirascpa-my.sharepoint.com/:f:/g/personal/mima619_cira_it/Ei3gkk07FENMmW2S9l-jWr0BXNRhm4rBquPvHPUBcJaNOQ?e=dBbhB6

Two different formats of the grids are included: CGNS and native SU2. However the problem was encountered with both files.
I used the last SU2 code version (8.0.0), on a Linux HPC cluster with MPI.

Could anyone provide suggestions to solve this issue?
Thanks for your support.
Kind Regards,
Mauro Minervino

**Comment by rois1995** [comment]
Hi Mauro,

I know that this Discussion is old, but I'll answer nevetheless.

I tried running with your setup and I found out that the y+ is larger than 1 in almost every point of the grid, this can be a problem when switching to a RANS simulation, and I suppose even more when using the SST model due to the boundary condition for Omega. Plus, it is not recommended to use the MUSCL_TURB option, instead just discretize the turbulence via a first order scheme. 

Also, I would not trust the results that you obtained with higher AoAs due to the problem with the y+ values.

Thus, try refining in the wall normal direction of at least a factor of 2 (I think also the wing surface needs more refinement due to the sharp trailing edge) and deactivate the MUSCL_TURB option.
